pyveg/scripts/analyse_gee_data.py

```diff
@@ -71,6 +71,7 @@ def main():
                       default="gee_img.png")
     parser.add_argument("--input_file",help="text file with coordinates, one per line")
     parser.add_argument("--mask_cloud",help="EXPERIMENTAL - apply cloud masking function",action='store_true')
+    parser.add_argument("--network_centrality",help="EXPERIMENTAL - apply cloud masking function",action='store_true')
```

Comment:
Should probably edit the "help" text here to explain what the network_centrality argument does.

Reply:
Ok! Will be done in this branch before merging.